udata/settings.py

```diff
@@ -37,6 +37,8 @@ class Defaults(object):
     CACHE_KEY_PREFIX = 'udata-cache'
     CACHE_TYPE = 'redis'
 
+    SERVER_NAME = 'localhost:7000'
```

Comment:
I'd also rather have the same config with `inv serve` and `manage.py serve` (ie debug)